.github/actions/shellcheck/action.yml

```diff
@@ -0,0 +1,64 @@
+# This is a reusable action for running ShellCheck,
+# a linter for shell scripts (https://shellcheck.net).
+
+# For more context, see:
+# https://github.com/openedx/.github/blob/master/docs/decisions/0001-shellcheck.rst
+
+name: ShellCheck
+
+# For details on each of these input arguments, see:
+#   ../../../workflow-templates/shellcheck.yml
+# which invokes this action.
+inputs:
```

Comment:
It turns out that reusable workflow can't accept arguments. They can accept secrets, but these aren't secrets.

So, I turned this into an action instead.

Reply:
I think they can accept both secrets and inputs: https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-inputs-and-secrets-in-a-reusable-workflow

(I still can't figure out the difference between actions and reusable workflows.)

Reply:
ARE YOU KIDDING ME

Reply:
From some more poking around, it seems there's a huge amount of overlap between Composite Actions and Reusable Workflows, and they appear to have converged over time. The actions seem to be more limited at this point—according to one outdated blog post, they cannot accept secrets, don't have conditionals or multiple jobs, and only produce a single log statement. (The last one is true, and the secrets thing looks true as well. But there was a claim that reusable workflows can't call each other, which is no longer true. So take the others with a grain of salt.) I think the only limitation on the reusable workflow side is that they can only call each other to a depth of 4, whereas actions can call each other to a depth of 10.

Reply:
Gosh, I like GHA, but I wish it and its documentation were more cohesive.

Thanks for the tips @timmc-edx .